docs/installation/installing-with-npm.md

```diff
@@ -127,9 +127,14 @@ Include the `node_modules/govuk-frontend/all.js` script on your page. You might
 
 To initialise all components, use the `initAll` function.
 
+JavaScript in GOV.UK Frontend requires the DOM to be parsed first by the browser before it is initialised. Because of this, make sure you include the script before the closing `</body>` tag.
```

Comment:
Not blocking:

I think the DOM is a parsed representation of the HTML document, so the browser parses the HTML into the DOM, rather than parsing the DOM directly?

Reply:
i've amended the wording